python/cuml/common/base.pyx

```diff
@@ -17,6 +17,7 @@
 # distutils: language = c++
 
 import inspect
+import nvtx
```

Comment:
We have a few legacy usages that probably should be removed/standardized to avoid confusions, for example: 
https://github.com/rapidsai/cuml/blob/31358cc48427cf776bdf719a5b45696a9f90de98/python/cuml/tsa/arima.pyx#L375
https://github.com/rapidsai/cuml/blob/31358cc48427cf776bdf719a5b45696a9f90de98/python/cuml/naive_bayes/naive_bayes.py#L247